### PR TITLE
Add hERG Central with multiple target

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

File: data/hERG_Central/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: hERG Central
+description: "Human ether-\xE0-go-go related gene (hERG) is crucial for the coordination\
```
Comment (kjappelbaum) [inline data/hERG_Central/meta.yaml:2]: shall we use the unicode symbol here instead of \xE0?

**Comment by kjappelbaum** [comment]
Is this partially the same as #53 ?

**Review by phalem — COMMENTED** [review]
Remove Character with acronym

**Review by phalem — COMMENTED** [review]
Add benchmark field

File: data/herg_central/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+name: herg_central
+description: "Human ether-à-go-go related gene (hERG) is crucial for the coordination\nof the heart's beating. Thus, if a drug blocks the hERG, it could\
+    \ lead to severe\nadverse effects. Therefore, reliable prediction of hERG liability in the early\nstages of drug design is quite important to reduce\
+    \ the risk of cardiotoxicity-related\nattritions in the later development stages. There are three targets: hERG_at_1microM,\nhERG_at_10microM, and hERG_inhib."
```
Comment (kjappelbaum) [inline data/herg_central/meta.yaml:5]: same question about `\`

File: data/herg_central/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      description: the percent inhibition of hERG at a 1uM concentration
+      units: uM
+      type: continuous
+      names:
+          - noun: hERG percent inhibition at a 1uM concentration
+          - noun: hERG percent inhibition at 1uM
+          - noun: human ether-à-go-go related gene (hERG) percent inhibition at a 1uM concentration
+          - noun: human ether-à-go-go related gene (hERG) percent inhibition at 1uM
+          - verb: active against hERG at a 1uM concentration
+          - verb: active against human ether-à-go-go related gene (hERG) at a 1uM concentration
```
Comment (kjappelbaum) [inline data/herg_central/meta.yaml:17]: do we need to include something like `is`?

File: data/herg_central/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - id: hERG_at_1uM
+      description: the percent inhibition of hERG at a 1uM concentration
+      units: uM
+      type: continuous
+      names:
+          - noun: hERG percent inhibition at a 1uM concentration
+          - noun: hERG percent inhibition at 1uM
+          - noun: human ether-à-go-go related gene (hERG) percent inhibition at a 1uM concentration
+          - noun: human ether-à-go-go related gene (hERG) percent inhibition at 1uM
+          - verb: active against hERG at a 1uM concentration
```
Comment (kjappelbaum) [inline data/herg_central/meta.yaml:16]:
```suggestion
          - verb: is active against hERG at a 1uM concentration
```

File: data/herg_central/meta.yaml (hunk 3 above)
Comment (kjappelbaum) [inline data/herg_central/meta.yaml:17]:
```suggestion
          - verb: is active against human ether-à-go-go related gene (hERG) at a 1uM concentration
```

**Review by MicPie — APPROVED** [review]

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**